CONTRIBUTING.md

```diff
@@ -42,8 +43,9 @@ pipeline to check for common errors and possible bugs.
 ## Run this application
 
 Now that your environment is all set up, we can run the application. 
-You can get it up and running by typing `npm run dev` in the project
-root directory.
+To do that, first you need to install required dependencies. Run
+`yarn install` and once is finished, you can get it up and running
+by typing `npm run dev` in the project root directory.
```

Comment:
According to our updated `package.json` file, `npm run dev` shouldn't work anymore. We must follow the above suggestion (using yarn) or use `npm run start` instead.

Reply:
Updating to use yarn to keep the same pattern.